src/store/model/megekko.ts

```diff
@@ -0,0 +1,159 @@
+import {Store} from './store';
+
+export const Megekko: Store = {
+	labels: {
+		inStock: {
+			container: '.product-order .text_green',
+			text: ['dag', 'werkdag']
+		},
+		maxPrice: {
+			container: '.col_right_container .euro',
+			euroFormat: false
+		},
+		outOfStock: {
+			container: '.product_detail .text_red',
+			text: ['minimaal 10 dagen']
+		}
+	},
+	links: [
+		{
+			brand: 'test:brand',
+			model: 'test:model',
+			series: 'test:series',
+			url: 'https://www.coolblue.nl/product/826844/'
```

Comment:
test link is still for cool blue, please use something from megekko

Reply:
ugh... my bad, will fix right now.